### ISSUE TITLE
 fonts not displayed correctly / email content encoding issues

### ISSUE THREAD

**Issue by fuorigioco1981** [thunderbird/thunderbird-android#8393]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

8.0b4

### Where did you get the app from?

Google Play

### Android version

14

### Device model

Samsung s21 fe 

### Steps to reproduce

 email open
then fonts don't display correctly

aquamail, BlueMail shows the same email correctly 

### Expected behavior

show fonts

### Actual behavior

fonts not displayed 
![Screenshot_20241023_090102_Aqua Mail](https://github.com/user-attachments/assets/f33e787d-75d0-40fa-8e6d-b3f067578def)
![Screenshot_20241023_090005_Thunderbird Beta](https://github.com/user-attachments/assets/df9a1bab-b98e-4eb8-8798-9da548f338df)

**Comment by fuorigioco1981** [comment]
any help from the beta channel to fix this?

**Comment by kewisch** [comment]
Can you provide the full source of that email? You might need to retrieve it from another client. We adhere to the character set that the email provides, if this was set wrong by the sender we may be showing the wrong characters. Other clients may be ignoring the standards here and trying their best to display it anyway.

**Comment by fuorigioco1981** [comment]
at the moment i retrieve emails from 7 accounts from thunderbird, aquamail and BlueMail (till i stay with thunderbird only of course). the other 2 display the same email correctly
how do i provide the full source of the email?

**Comment by kewisch** [comment]
The easiest way to do this is via Desktop, is that option for you? https://github.com/thunderbird/thunderbird-android/issues/1018

**Comment by fuorigioco1981** [comment]
> The easiest way to do this is via Desktop, is that option for you? https://github.com/thunderbird/thunderbird-android/issues/1018

yes i can try. how do i do it?

**Comment by fuorigioco1981** [comment]
i can't do the thing with finding the source etc..
the quota why the same email in thunderbird desktop fonts appear ok but on thunderbird android not?

**Comment by kewisch** [comment]
Use Ctrl/Cmd+U when you are on the email list.

**Comment by fuorigioco1981** [comment]
iam talking about thunderbird android .....

**Comment by kewisch** [comment]
Yes, though you can't view source on Thunderbird Android. As I mentioned, you'd need to use Thunderbird Desktop for this.
